Docker-compose/run-docker-compose.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
-export COMPOSE_PROFILES=demo
+export COMPOSE_PROFILES=wwwallet
```

Comment:
I would keep demo only

wwwallet would be set by hand

Reply:
otherwise, use another run script that do everything needed for wwwallet demo and therefore runs the demo one

Reply:
@PascalDR this doesn't seem to be resolved yet